lib/switchtec.c

```diff
@@ -880,26 +964,45 @@ static int write_parsed_log(struct log_a_data log_data[],
 		hours = time % 24;
 		days = time / 24;
 
-		/*
-		 * log entry number, module ID, and log severity are in the
-		 * 3rd DWord
-		 */
-		entry_num = log_data[i].data[2] & 0x0000FFFF;
-		mod_id = (log_data[i].data[2] >> 16) & 0xFFF;
-		log_sev = (log_data[i].data[2] >> 28) & 0xF;
-
-		/* validate the module ID */
-		if ((mod_id > defs->num_alloc) ||
-		    (strlen(defs->module_defs[mod_id].mod_name) == 0)) {
-			if (fprintf(log_file, "(Invalid module ID: 0x%x)\n",
-				    mod_id) < 0)
-				goto ret_print_error;
-			continue;
+		if (log_type == SWITCHTEC_LOG_PARSE_TYPE_APP) {
+			/*
```

Comment:
So why is the functionality changing in this branch? That's not expected. If you want to change the functionality of APP logs, it should be done first in another patch with a commit message that tells me why we are making that change.

Reply:
The functionality isn't changing for app logs. The parsing is just reorganized since some of it doesn't apply to mailbox logs.